minidump-stackwalk/stackwalker.cc

```diff
@@ -795,6 +795,35 @@ static string stripquotes(const string& s) {
   return s;
 }
 
+static string trim(const string& s) {
+  size_t first = s.find_first_not_of(" \t");
+  if (first == string::npos) {
+    first = 0;
+  }
+
+  size_t last = s.find_last_not_of(" \t");
+  if (last == string::npos) {
+    last = s.length();
+  }
+
+  return s.substr(first, last);
+}
+
+static void ConvertCPUInfoToJSON(const string& cpuinfo,
+                                 Json::Value& root) {
+  for (string& line : split(cpuinfo, '\n')) {
+    vector<string> bits = split(line, ':');
+    if (bits.size() != 2 || !startswith(bits[0], "microcode")) {
+        continue;
+    }
+
+    try {
+        root["system_info"]["cpu_microcode_version"] = static_cast<Json::UInt>(std::stoi(trim(bits[1]), 0, 0));
```

Comment:
two nits:
1. You should pass `nullptr` as the second parameter here
2. I think you should explicitly pass `16` for the third parameter, since this should always be a hex string. 

Adding C++ exception usage to this file makes me a little uncomfortable, since we're not currently using them elsewhere, but this is only ever run as a standalone binary anyway so I think it should be fine. (We just don't use C++ exceptions anywhere in Gecko so I have a bit of an aversion to them.)